lib/rex/post/meterpreter/extensions/stdapi/sys/registry.rb

```diff
@@ -272,6 +334,24 @@ def Registry.enum_value(hkey)
     return values
   end
 
+  def Registry.enum_value_direct(root_key, base_key, perm = KEY_READ)
+    request = Packet.create_request('stdapi_registry_enum_value_direct')
+    values  = []
+
+    request.add_tlv(TLV_TYPE_ROOT_KEY, root_key)
+    request.add_tlv(TLV_TYPE_BASE_KEY, base_key)
+    request.add_tlv(TLV_TYPE_PERMISSION, perm)
+
+    response = client.send_request(request)
+
+    response.each(TLV_TYPE_VALUE_NAME) { |value_name|
+      values << Rex::Post::Meterpreter::Extensions::Stdapi::Sys::RegistrySubsystem::RegistryValue.new(
+          client, 0, value_name.value)
+    }
+
+    return values
```

Comment:
You can omit the return keyword here https://github.com/bbatsov/ruby-style-guide#no-explicit-return